.gitignore

```diff
@@ -106,6 +106,10 @@ venv.bak/
 
 # custom
 mmpose/version.py
+mmpose/configs
```

Comment:
Is it intended?

Reply:
Yes. To support MIM, we need to create these symlinks in the folder "mmpose" when installing mmpose, which should not be tracked by git.

Reply:
Yes. To support MIM, we need to create these symlinks in the folder "mmpose" when installing mmpose, which should not be tracked by git.

Reply:
Alternatively, we can remove these links at the end of setup.py.

Reply:
It is better to delete them afterwards.

Reply:
On second thought, I guess removing the symlinks may cause errors when the mmpose is installed from the source code instead of pip.